Sources/MuxUploadSDK/Upload/ChunkedFileUploader.swift

```diff
@@ -88,7 +88,7 @@ class ChunkedFileUploader {
         let task = Task.detached { [self] in
             do {
                 // It's fine if it's already open, that's handled by ignoring the call
-                let fileSize = file.fileSize
```

Comment:
This gives us two sources of file size, so I think we should unify them. 

`ChunkedFile.fileSize` is a computed property. Can you either back it with `fetchFileSize()` or have `fetchFileSize()` cache the value it fetches. Personally I'd prefer the former.

Reply:
Also, does this build for you? I get `cannot find 'inputFileURL' in scope`

Reply:
The file size remains constant over the course of the upload, so I don't expect the value to change.

(It it did, I don't even think that GCP would accept the new size in the `Content-Range` header, so this would be circumstance would be hella exceptional)

Reply:
> Also, does this build for you? I get `cannot find 'inputFileURL' in scope`

Bad cherry pick from my working branch, fixed in [0f668dd](https://github.com/muxinc/swift-upload-sdk/pull/50/commits/0f668ddc6e96aea4282e6fc3c1ad3528a2b989ef)

Reply:
There's a potential issue where if the `fileSize` property is accessed before the file is actually opened, it will return the wrong value. In a sense the original implementation had a similar issue to mine since the file size was stored in two places. 

(One being the iOS file system itself and the other being this property.)